cmd/cluster-kube-scheduler-operator/render/config.go

```diff
@@ -0,0 +1,30 @@
+package render
+
+type Config struct {
+	// ConfigHostPath is a host path mounted into the scheduler pods to hold the config file.
+	ConfigHostPath string
+
+	// ConfigFileName is the filename of config file inside ConfigHostPath.
+	ConfigFileName string
+
+	// CloudProviderHostPath is a host path mounted into the apiserver pods to hold cloud provider configuration.
```

Comment:
Do we need this field?

Reply:
i think no, still need to clean up some stuff.